### ISSUE TITLE
Restart/Shutdown acts on the entire server, not just Umbrel

### ISSUE THREAD

**Issue by thefeli73** [getumbrel/umbrel#781]
It is reasonable to expect that the Restart/Shutdown buttons Restart/Shutdown your machine if you're running UmbrelOS, however this was very unexpected when running on a server and you just want to restart Umbrel, not the machine running dozens of other services. Either change the button descriptions to make it obvious what they do or change the behaviour to Restart/Shutdown Umbrel only, at least when not running on umbrelOS

**Comment by pjw65** [comment]
Yes I was shocked when I pressed the stop button and my linux server shut down.

The `scripts/stop` only stops umbrel.
